src/Console/Application.php

```diff
@@ -286,10 +290,19 @@ public function buildContainer()
         };
 
         // VIOLATION RENDERER
-        $c['violation.renderer'] = function ($c) {
+        $c['violation.renderer.console'] = function ($c) {
             return new Renderer\ConsoleOutputRenderer(new ConsoleOutput(), $c['violation.message_helper']);
         };
 
+        $c['violation.renderer.html'] = function ($c) {
+            return new Renderer\HtmlOutputRenderer(
+                new ConsoleOutput(),
```

Comment:
_If_ you really want to do console output inside your class, you should reuse the `OutputInterface` instance the command receives. Of course, that means you cannot use DI to inject it.